v2/elasticsearch-common/src/main/java/com/google/cloud/teleport/v2/elasticsearch/transforms/WriteToElasticsearch.java

```diff
@@ -69,7 +69,7 @@ public static Builder newBuilder() {
   private static final String DOCUMENT_TYPE = "_doc";
 
   @Override
-  public PDone expand(PCollection<String> jsonStrings) {
+  public PCollection<String> expand(PCollection<String> jsonStrings) {
```

Comment:
Since this function uses [ElasticsearchIO.write()](https://beam.apache.org/releases/javadoc/2.34.0/org/apache/beam/sdk/io/elasticsearch/ElasticsearchIO.Write.html) which still returns PDone can you explain how can we get  PCollection<String>

Reply:
We get PCollection in modified ElasticsearchIO